migrations/Version20240124152500.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240124152500 extends MSPMigration
+{
+    public function getDescription(): string
+    {
+        return 'Simplifies Server Manager';
+    }
+
+    protected function getDatabaseType(): ?MSPDatabaseType
+    {
+        return new MSPDatabaseType(MSPDatabaseType::DATABASE_TYPE_SERVER_MANAGER);
+    }
+
+    protected function onUp(Schema $schema): void
+    {
+        // phpcs:ignoreFile Generic.Files.LineLength.TooLong
+        $this->addSql('ALTER TABLE game_list CHANGE password_player password_player LONGTEXT DEFAULT NULL, CHANGE api_access_token api_access_token VARCHAR(32) DEFAULT NULL, CHANGE server_version server_version VARCHAR(45) DEFAULT NULL');
+        $this->addSql('ALTER TABLE game_saves CHANGE save_timestamp save_timestamp TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL');
+    }
+
+    protected function onDown(Schema $schema): void
+    {
+        // phpcs:ignoreFile Generic.Files.LineLength.TooLong
```

Comment:
Since the phpcs:ignoreFile is targeting the file , I think only one line is needed.  You could even put it on the first line of the file I think.
```suggestion
```

Reply:
ok, so the password_player, api_access_token and server_version are allowed NULL now , 
and save_timestamp is a DATETIME now . That last one is because doctrine works mostly with DATETIME ? There might be ways to support TIMESTAMP as well , but this might be the easiest way if it does not have large consequences.  the server manager db is only used by the server manager itself , so it should not impact to many code